ZeroWasteFriendlyTests/EcoMap/FakeEcoMapPresenter.swift

```diff
@@ -0,0 +1,52 @@
+//
+//  FakeEcoMapPresenter.swift
+//  ZeroWasteFriendly
+//
+//  Created by Marek Skrzelowski on 20/02/2020.
+//  Copyright © 2020 vandermesis. All rights reserved.
+//
+
+import UIKit
+
+@testable import ZeroWasteFriendly
+
+final class FakeEcoMapPresenter: EcoMapPresenter {
```

Comment:
we're doing all that stuff with generic shared presenter etc, to not need to write in every presenter logic for spinners, alerts and errors

you can make your own TESTSharedPresenter that is having all these, and you'll not need to implement it everywhere